doc/prepare_raspberrypi.rst

```diff
@@ -291,3 +291,26 @@ And to turn it off::
 Note that you need root privileges to control the LEDs.
 
 Similarly for the red (power) LED, which is called ``led1``.
+
+
+Advanced users only: Control Linux real-time properties
+-------------------------------------------------------
+Add this to the first (and only) line in ``/boot/cmdline.txt``::
+
+   isolcpus=2
+
+Run the sample application on a specific CPU core, by modifying the
+autostart file ``/lib/systemd/system/pnet-sampleapp.service`` (if installed)::
+
+   ExecStart=taskset -c 2 /home/pi/profinet/build/pn_dev -v -b /sys/class/gpio/gpio22/value -d /sys/class/gpio/gpio27/value
+
+Display which CPU core a process is running on::
+
+   pi@pndevice-pi:~$ taskset -c -p $(pidof pn_dev)
+   pid 443's current affinity list: 2
+
+Display real time properties of a process::
+
+   pi@pndevice-pi:~$ chrt -p $(pidof pn_dev)
+   pid 438's current scheduling policy: SCHED_OTHER
```

Comment:
An advanced user should maybe use SCHED_FIFO instead?

Reply:
Changed to 
```
Display real time properties of a process (should typically be ``SCHED_FIFO``
for best result)::
```
Done